.github/workflows/ci.yml

```diff
@@ -38,3 +38,11 @@ jobs:
       env:
         TOXENV: ${{ matrix.toxenv }}
       run: tox
+    - name: Run check-keywords
```

Comment:
I wonder if we should add a comment mentioning why this is being added and what to do if you add a keyword and see an error?

Reply:
thanks, @adzuci  I've added a comment here.